modules/ROOT/examples/live-demos/full-featured/example.js

```diff
@@ -148,5 +148,6 @@ tinymce.init({
       value: 'Salutation',
       title: 'Salutation'
     }
-  ]
+  ],
+  revisionhistory_fetch: fetch_revisions,
```

Comment:
This is undefined. This option needs to be implemented correctly for users who just copy paste.

Reply:
[previous code](https://github.com/tinymce/tinymce-docs/pull/3161/files#diff-c285688d2a88fec7c645a37f976828932daa9360a988e4c83f6946ed69a7e74eR115-R118) in `example.js` contains other incomplete function stubs. I have added a comment in [363e718](https://github.com/tinymce/tinymce-docs/pull/3161/commits/363e7180e507f6d73b8120da7ea433e11f567cea) to make it obvious that the user needs to implement these functions. however, I would agree it makes sense to include at least a basic implementation of the function so that users who want to copy-paste the code have an easier time getting started.

Reply:
@MitchC1999 [previous code](https://github.com/tinymce/tinymce-docs/pull/3161/files#diff-c285688d2a88fec7c645a37f976828932daa9360a988e4c83f6946ed69a7e74eR115-R118) in `example.js` contains other incomplete function stubs. I have added a comment in [363e718](https://github.com/tinymce/tinymce-docs/pull/3161/commits/363e7180e507f6d73b8120da7ea433e11f567cea) to make it obvious that the user needs to implement these functions. however, I would agree it makes sense to include at least a basic implementation of the function so that users who want to copy-paste the code have an easier time getting started.

Reply:
even something like `revisionhistory_fetch: () => Promise.resolve([])` is enough.